src/components/backend-ai-session-list.ts

```diff
@@ -2928,6 +2928,7 @@ ${rowData.item[this.sessionNameField]}</pre
                   class="fg blue controls-running"
                   id="${rowData.index + '-assignment'}"
                   icon="assignment"
+                  ?disabled="${rowData.item.status === 'CANCELLED'}"
```

Comment:
This is a minor thing, but I think we shall use triple equal (`===`) instead of `==`. Please refer [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Equality_comparisons_and_sameness#strict_equality_using).

Reply:
And I just changed the operator to triple equal sign.